src/views/ideas/ideas.scss

```diff
@@ -17,127 +17,133 @@ $base-bg: $ui-white;
     background: bottom left url("/images/ideas/left-juice.png") no-repeat;
 }
 
-.ttt-section {
-    display: flex;
-    margin: 0 auto;
-    text-align: center;
-    justify-content: center;
-    flex-wrap: wrap;
-    align-items: center;
-}
-
-.tips-divider {
-    border-top: 1px solid $ui-gray;
-    width: 100%;
-}
-
-.tips-banner-image {
-    max-width: calc(100% - 2rem);
-}
-
-.banner-button {
-    background-color: $ui-white;
-    color: $ui-aqua;
-    font-size: 1rem;
```

Comment:
Were these styles intentionally removed? `ideas.jsx` still references this class (line 104).

Reply:
Yes, the intent for this task was to redesign the header as well but the content team delayed providing some links so we will handle it separately.